web/client/epics/annotations.js

```diff
@@ -346,10 +346,12 @@ module.exports = (viewer) => ({
     setStyleEpic: (action$, store) => action$.ofType(SET_STYLE)
```

Comment:
the name of this epic is a little too generic. Unfortunately we have global names for epics. I suggest to change it with `setAnnotationStyle`

Reply:
ok